dev-ml/ocaml-gettext/ocaml-gettext-0.4.2-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DUNE_PKG_NAME="gettext"
+inherit dune
+
+DESCRIPTION="Provides support for internationalization of OCaml program"
+HOMEPAGE="https://github.com/gildor478/ocaml-gettext"
+SRC_URI="https://github.com/gildor478/ocaml-gettext/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1-with-linking-exception"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="+ocamlopt test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	>=dev-ml/cppo-1.6.6
+	dev-ml/dune-configurator
+"
+RDEPEND="
+	dev-ml/base:=
+	<dev-ml/camomile-2.0.0:=[ocamlopt=]
```

Comment:
Maybe in a new revision we should look at https://github.com/gildor478/ocaml-gettext/issues/14? Maybe put a comment about it for now.

Reply:
ok

Reply:
should we file a bug for it?

Reply:
*the ideal* solution would be to split `ocaml-gettext` like upstream does... also maybe remove gettext-camomile completely if we do not need it?

Reply:
> _the ideal_ solution would be to split `ocaml-gettext` like upstream does... also maybe remove gettext-camomile completely if we do not need it?

Actually... we already have this split! just the ocaml-gettext pkg is badly written by requiring  camomile

Reply:
> 

So we have to fix ocaml-gettext and move the pin to dev-ml/ocaml-gettext-camomile.

Reply:
Done. I went with the Fedora hack of just removing things that would otherwise require camomile.